src/api/app/views/webui2/webui/project/_monitor_control.html.haml

```diff
@@ -7,35 +7,44 @@
         %button.btn.btn-outline-secondary.dropdown-toggle{ data: { toggle: :dropdown }, type: :button }
           %span.caret
           Status
-        .dropdown-menu
-          %button.btn.btn-link.monitor-no-filter-link{ type: :button } No filter
+        .dropdown-menu.keep-open
+          .text-center
+            %button.btn.btn-link.monitor-filter-link{ type: :button } All
+            %button.btn.btn-link.monitor-no-filter-link{ type: :button } None
           - status.each do |status|
-            .custom-control.custom-checkbox.dropdown-item.ml-2
-              = check_box_tag valid_xml_id(status), 1, status_filter.include?(status), class: 'custom-control-input'
-              %label.custom-control-label{ for: valid_xml_id(status) }
+            .dropdown-item
+              .custom-control.custom-checkbox
+                = check_box_tag valid_xml_id(status), 1, status_filter.include?(status), class: 'custom-control-input'
+                %label.custom-control-label{ for: valid_xml_id(status) }
```

Comment:
Clicking on the label doesn't check/uncheck the checkboxes in the `Status` dropdown. The indentation of the line below needs to be one level deeper.

Just to be extra clear, right now, it's:
```
%label
= status
```

It should be:
```
%label
  = status
```
``